index.js

```diff
@@ -15,8 +15,18 @@ if (hasFlag('no-color') ||
 	hasFlag('color=always')) {
 	forceColor = true;
 }
+let forceLevel;
 if ('FORCE_COLOR' in env) {
-	forceColor = env.FORCE_COLOR.length === 0 || parseInt(env.FORCE_COLOR, 10) !== 0;
+	if (env.FORCE_COLOR === true || env.FORCE_COLOR === 'true') {
+		forceColor = true;
+	} else if (env.FORCE_COLOR === false || env.FORCE_COLOR === 'false') {
+		forceColor = false;
+	} else {
+		forceLevel = env.FORCE_COLOR.length === 0 ?
+			1 :
+      parseInt(env.FORCE_COLOR, 10);
+		forceColor = forceLevel !== 0;
+	}
```

Comment:
You can simplify this to just using `forceColor` with a numeric level. `true` would make it `1` to be backwards compatible, `false` would make it 0, and `2` or `3` specifically would make it the respective level.

Then wherever we check `forceColor`, change _that_ behavior to return the value instead. Would be a simpler way to do it instead of messing around with two variables.